src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -99,74 +66,106 @@ impl SinkConfig for StackdriverConfig {
 struct HttpEventSink {
     config: StackdriverConfig,
     started: DateTime<Utc>,
+    client: reqwest::Client,
```

Comment:
Our standard practice has been to use `hyper` directly for HTTP clients, and we would prefer to continue that pattern.